@noun-auction/components/DataRenderers/AuctionBidder.tsx

```diff
@@ -69,17 +65,17 @@ export function AuctionBidder({
           style={{ lineHeight: '1.15' }}
           align="right"
           className={[sideBarUpperLabel]}
-          color={auctionData?.rpcData?.bidder !== AddressZero ? 'primary' : 'tertiary'}
+          color={highestBidder !== AddressZero ? 'primary' : 'tertiary'}
```

Comment:
Would recommend using !isAddressMatch(highestBidder, AddressZero) for readability